egs/librimix/ConvTasNet/eval.py

```diff
@@ -58,6 +160,8 @@ def main(conf):
         sample_rate=conf["sample_rate"],
         n_src=conf["train_conf"]["data"]["n_src"],
         segment=None,
+        return_id=True,
+        # FIXME: ensure max mode for eval.
```

Comment:
New problem: we need the "max" version for ASR, but the other metrics should still be computed with the "min" mode so that the results are consistent with what we had before. 
We'll probably need to change the dataloader to return the lengths, other solutions?

Reply:
I would use max version all along. Otherwise you would not be able to say plot SI-SDR vs WER and compare the two directly.

Reply:
Yes, but SISDR results are usually reported on min, we cannot change that without the user knowing it. Otherwise recipes inspired by this might report wrong results afterward..